packages/js/test/plugins/auctionHouseModule/findBidsByPublicKeyField.test.ts

```diff
@@ -0,0 +1,160 @@
+import test from 'tape';
+import { sol } from '@/types';
+
+import {
+  killStuckProcess,
+  metaplex,
+  createNft,
+  createWallet,
+} from '../../helpers';
+import { createAuctionHouse } from './helpers';
+import { LazyBid } from '@/plugins';
+
+killStuckProcess();
+
+test('[auctionHouseModule] find all lazy bids by buyer', async (t) => {
```

Comment:
On each of these tests, I like that you create 2 listings/bids/purchases to ensure both are picked up but could you also add a 3rd one that should not be picked up to ensure that we are indeed filtering out the accounts we don't want?

Reply:
Thanks, updated!